internal/api/apiinternal/bootstrap.go

```diff
@@ -3,7 +3,7 @@ package apiinternal
 import "net/http"
 
 func (s *Server) Bootstrap(w http.ResponseWriter, r *http.Request) {
-	// do the actual bootstrap retrieval here
+	// do the actual bootstrap retrieval here along with token header enforcement
```

Comment:
Nit, but it's helpful to prefix these with `TODO:` for finding later, some editors will even pick up that syntax for making a TODO list.